setup.py

```diff
@@ -64,7 +64,8 @@
         "scikit-learn",
         "scipy",
         "pytest",
-        "transformers>=3.4,<3.6",
+        "transformers>=3.4,<4.1",
+        "sentencepiece",
```

Comment:
As per the [release notes](https://github.com/huggingface/transformers/releases/tag/v4.0.0), `sentencepiece` is not required as a dependency in `transformers` by default. But we use certain tokenizers from HF that require it.